@kiva/kv-components/src/vue/KvCartModal.vue

```diff
@@ -64,9 +64,10 @@
 
 						<!-- body -->
 						<div
+							v-if="showContentSlot"
```

Comment:
What situation would this not be the case?

Reply:
Not My Kiva case:
<img width="403" alt="Screenshot 2025-03-11 at 3 06 13 p m" src="https://github.com/user-attachments/assets/faa56d48-7eed-403a-86a6-6f5b5d8627c7" />